yb-voyager/cmd/exportSchema.go

```diff
@@ -766,9 +766,13 @@ func applyIndexFileTransformations() (*sqltransformer.IndexFileTransformer, erro
 	//assuming that assessment is run and fetched the redundant indexes
 	//TODO: see if we need to take care of the scenario where assessment is unable to fetch these
 	var err error
-	var redundantIndexToResolvedExistingIndex map[string]string
+	var redundantIndexToResolvedExistingIndex *utils.StructMap[*sqlname.ObjectNameQualifiedWithTableName, string]
 	redundantIndexToResolvedExistingIndex, err = fetchRedundantIndexMapFromAssessmentDB()
 	if err != nil {
+		if skipPerfOptimizations {
+			log.Infof("skipping error while fetching redundant index map from assessment db: %v", err)
+			return nil, nil
+		}
 		return nil, fmt.Errorf("failed to fetch redundant index map from assessment db: %w\n%s", err, sqltransformer.SUGGESTION_TO_USE_SKIP_PERF_OPTIMIZATIONS_FLAG)
 	}
 	indexTransformer := sqltransformer.NewIndexFileTransformer(redundantIndexToResolvedExistingIndex, bool(skipPerfOptimizations), source.DBType)
```

Comment:
same question as above: do we really need to pass `skipPerfOptimizations` to the transformers.